<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create distance distribution</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonInFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Network file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInFile">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonZoom">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>310</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>441</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOutFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Distribution file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutFile">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ncpu">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>250</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ncpu">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of CPUs</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_sphere">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>221</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="labelRadius">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Radius (um)</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCentre">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Centre (um)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditX0">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditZ0">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSphere">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Use spherical subregion?</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditY0">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRadius">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelZ0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>z0</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelX0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>x0</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelY0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>y0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxImage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_threshold">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Threshold distance</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditThreshold">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>60</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Image hypoxic regions?</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonTiffFile">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiff  file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTiffFile">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxSampling">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QCheckBox" name="checkBoxRandom">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Random sampling points?</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditGrid_dx">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelGrid_dx">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Grid spacing (um)</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelResultImage">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>963</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonInFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>inFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonZoom</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>distancer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxSphere</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>sphereOption()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonOutFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>outFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxRandom</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>randomOption()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonTiffFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>tiffFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxImage</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>imageOption()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>51</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonClickHandler()</slot>
  <slot>inFileSelecter()</slot>
  <slot>outFileSelecter()</slot>
  <slot>radioButtonHandler()</slot>
  <slot>distancer()</slot>
  <slot>sphereOption()</slot>
  <slot>randomOption()</slot>
  <slot>tiffFileSelecter()</slot>
  <slot>imageOption()</slot>
 </slots>
</ui>
